gtest/src/program.rs

```diff
@@ -172,3 +172,47 @@ impl<'a> Program<'a> {
         self.id
     }
 }
+
```

Comment:
Considering now about should be `gtest` unit tests used as core-logic-testing

Reply:
new issue will be created